projectile.el

```diff
@@ -1422,7 +1445,12 @@ IGNORED-DIRECTORIES may optionally be provided."
   "Determine which external command to invoke based on the project's VCS.
 Fallback to a generic command when not in a VCS-controlled project."
   (pcase vcs
-    ('git projectile-git-command)
+    ('git (if projectile-git-use-fd
```

Comment:
I think this should also be guarded by `executable-find`, so this won't fail if `fg` is not installed.

Reply:
https://github.com/bbatsov/projectile/pull/1825/commits/e49fee486cbb1e3d8caf446c91d057a7ffcb0f93 - guarding by the value of `projectile-fd-executable`, which caches the return value of `executable-find`.